### ISSUE TITLE
Improve "scroll to top" navigation in reading interface

### ISSUE THREAD

**Issue by SevenSidedSquare** [pressbooks/ideas#375]
### Feature Description

The scroll to top button in the reading interface can be distracting in the middle of the screen. Suggest placing it to the side.

**Comment by SteelWagstaff** [comment]
@MichelleWeremczuk i think this is probably something where your opinion now carries the most weight.

**Comment by MichelleWeremczuk** [comment]
I agree it’s distracting, mainly because it’s always visible while scrolling down. We could make it visible only when users scroll up, when it’s most needed. Placing it on the right also follows [good design practices](https://www.nngroup.com/articles/back-to-top/). Alternatively, we could remove it entirely, as improved in-page linking on the left would make the button redundant.

**Comment by arzola** [comment]
This issue has been automatically closed as part of an organization-wide backlog review.

**Reason:** This issue was created 4y 10m ago with no updates in the last 1y 2m.

If this is still relevant, please feel free to reopen it with updated context. We appreciate your contributions and want to ensure our issue tracker stays actionable.

**Learn more:** https://pressbooks.community/t/announcement-automatic-closure-of-old-github-tickets/4542
